rules-emerging-threats/2025/Exploits/CVE-2025-33053/proc_creation_win_exploit_cve_2025_33053.yml

```diff
@@ -0,0 +1,56 @@
+title: Potential Exploitation of RCE Vulnerability CVE-2025-33053
+id: abe06362-a5b9-4371-8724-ebd00cd48a04
+related:
+    - id: 9a2d8b3e-f5a1-4c68-9e21-7d9e1cf8a123
+      type: similar
+    - id: 04fc4b22-91a6-495a-879d-0144fec5ec03
+      type: similar
+status: experimental
+description: |
+    Detects potential exploitation of Remote Code Execution Vulnerability CVE-2025-33053
+    which involves unauthorized code execution via WebDAV through external control of file names or paths.
+    The exploit abuses legitimate utilities like iediagcmd.exe or CustomShellHost.exe  by manipulating
+    their working directories to point to attacker-controlled WebDAV servers, causing them to execute
+    malicious executables (like route.exe) from the WebDAV path instead of legitimate system binaries
+    through Process.Start() search order manipulation.
```

Comment:
```suggestion
    Detects potential exploitation of remote code execution vulnerability CVE-2025-33053
    which involves unauthorized code execution via WebDAV through external control of file names or paths.
    The exploit abuses legitimate utilities like iediagcmd.exe or CustomShellHost.exe  by manipulating
    their working directories to point to attacker-controlled WebDAV servers, causing them to execute
    malicious executables (like route.exe) from the WebDAV path instead of legitimate system binaries
    through Process.Start() search order manipulation.
```